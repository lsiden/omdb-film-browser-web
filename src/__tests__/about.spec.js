import React from "react"
import { shallow } from "enzyme"

import About from "about"

const createWrapper = () => {
  return shallow(<About />)
}

test("OmdbMain", () => {
  expect(createWrapper()).toMatchSnapshot()
})
