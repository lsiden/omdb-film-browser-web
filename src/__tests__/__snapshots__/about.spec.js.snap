// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OmdbMain 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <About />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <AboutBanner />,
        <div
          style={
                    Object {
                              "marginLeft": 20,
                            }
          }
>
          <ReactMarkdown
                    escapeHtml={true}
                    renderers={Object {}}
                    skipHtml={false}
                    source="
          ## Written By
          Lawrence Siden
          
          [Westside Consulting LLC](http://westsideconsultingllc.com)
          
          [westsideconsultingllc@gmail.com](mailto:westsideconsultingllc@gmail.com)
          
          ## Credits
          [Open Movie Database API](https://www.omdbapi.com/)
          
          ## Source Code
          https://github.com/lsiden/omdb-film-browser-web
          "
                    transformLinkUri={[Function]}
          />
</div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <ReactMarkdown
            escapeHtml={true}
            renderers={Object {}}
            skipHtml={false}
            source="
## Written By
Lawrence Siden

[Westside Consulting LLC](http://westsideconsultingllc.com)

[westsideconsultingllc@gmail.com](mailto:westsideconsultingllc@gmail.com)

## Credits
[Open Movie Database API](https://www.omdbapi.com/)

## Source Code
https://github.com/lsiden/omdb-film-browser-web
"
            transformLinkUri={[Function]}
/>,
          "style": Object {
            "marginLeft": 20,
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "escapeHtml": true,
            "renderers": Object {},
            "skipHtml": false,
            "source": "
## Written By
Lawrence Siden

[Westside Consulting LLC](http://westsideconsultingllc.com)

[westsideconsultingllc@gmail.com](mailto:westsideconsultingllc@gmail.com)

## Credits
[Open Movie Database API](https://www.omdbapi.com/)

## Source Code
https://github.com/lsiden/omdb-film-browser-web
",
            "transformLinkUri": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <AboutBanner />,
          <div
            style={
                        Object {
                                    "marginLeft": 20,
                                  }
            }
>
            <ReactMarkdown
                        escapeHtml={true}
                        renderers={Object {}}
                        skipHtml={false}
                        source="
            ## Written By
            Lawrence Siden
            
            [Westside Consulting LLC](http://westsideconsultingllc.com)
            
            [westsideconsultingllc@gmail.com](mailto:westsideconsultingllc@gmail.com)
            
            ## Credits
            [Open Movie Database API](https://www.omdbapi.com/)
            
            ## Source Code
            https://github.com/lsiden/omdb-film-browser-web
            "
                        transformLinkUri={[Function]}
            />
</div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <ReactMarkdown
              escapeHtml={true}
              renderers={Object {}}
              skipHtml={false}
              source="
## Written By
Lawrence Siden

[Westside Consulting LLC](http://westsideconsultingllc.com)

[westsideconsultingllc@gmail.com](mailto:westsideconsultingllc@gmail.com)

## Credits
[Open Movie Database API](https://www.omdbapi.com/)

## Source Code
https://github.com/lsiden/omdb-film-browser-web
"
              transformLinkUri={[Function]}
/>,
            "style": Object {
              "marginLeft": 20,
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "escapeHtml": true,
              "renderers": Object {},
              "skipHtml": false,
              "source": "
## Written By
Lawrence Siden

[Westside Consulting LLC](http://westsideconsultingllc.com)

[westsideconsultingllc@gmail.com](mailto:westsideconsultingllc@gmail.com)

## Credits
[Open Movie Database API](https://www.omdbapi.com/)

## Source Code
https://github.com/lsiden/omdb-film-browser-web
",
              "transformLinkUri": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
